sklearn/preprocessing/label.py

```diff
@@ -209,7 +209,7 @@ class LabelEncoder(BaseEstimator, TransformerMixin):
     See also
     --------
     sklearn.preprocessing.OrdinalEncoder : encode categorical features
-        using a one-hot or ordinal encoding scheme.
+        using an ordinal encoding scheme.
```

Comment:
```suggestion
        Uses an ordinal encoding scheme.
```

Reply:
That line is a continuation of the above sentence which states:
> sklearn.preprocessing.OrdinalEncoder : encode categorical features

So using `Uses an ordinal encoding scheme.` it will be
> sklearn.preprocessing.OrdinalEncoder : encode categorical features Uses an ordinal encoding scheme.

Reply:
Ah I see okay